src/sentry/integrations/vercel/client.py

```diff
@@ -39,8 +43,22 @@ def get_user(self):
         return self.get_cached(self.USER_URL)["user"]
 
     def get_projects(self):
-        # TODO: we will need pagination since we are limited to 20
-        return self.get(self.PROJECTS_URL)["projects"]
+        limit = 20
+        params = {"limit": limit}
+        projects = []
+        # no one should have more than 200 projects
+        for i in range(10):
+            resp = self.get(self.PROJECTS_URL, params=params)
+            projects += resp["projects"]
+            # if we have less projects than the limit, we are done
+            if resp["pagination"]["count"] < limit:
```

Comment:
Hey can you link to the documentation? I want to double check why we'd want to use `resp["pagination"]["count"]` rather than `len(resp["projects"])`. Also why we pass `params = {"limit": limit}` if it's also always limited to 20.

Reply:
@mgaeta Docs here: https://vercel.com/docs/api#endpoints/projects/get-projects

I want to explicitly pass the limit so that the logic we have about the count versus the limit is not inferred.

Reply:
👍 Explicit is good.
`params["since"] = resp["pagination"]["next"] + 1` might miss projects if they're at the page border and have the same `updatedAt`, but the API doesn't have a way to mitigate that.

Reply:
@mgaeta Yea, that's theoretically possible but as you say there's not a good way we can handle that.